web/app/themes/dp-timber-theme/components/organisms/Header/Header.scss

```diff
@@ -0,0 +1,5 @@
+nav {
+	img {
+		width: 140px;
```

Comment:
can we add width attribute to img tag and see if we need this scss?

Reply:
Yes I can that too as well

Reply:
Yes I will do it